### PR TITLE
[WIP] Doc changes for Landr

### PR DESCRIPTION
**Pull request by tmigone**
Restructure Readme and docs to have them play nice with Landr.

### CODE REVIEW

File: meta.json [hunk 1]
```diff
@@ -0,0 +1,1362 @@
+{
```
Comment (LucianBuzzo) [inline meta.json:1]: landrbot will automatically generate this file, so it can be removed

Comment (tmigone) [reply meta.json:1]: sweet, will it be added to source control too? I guess it's not necessary

**Comment by AlexProgrammerDE** [comment]
@tmigone i think you can set this PR to a Draft.

**Comment by LucianBuzzo** [comment]
@AlexProgrammerDE AFAIK you can't turn a PR into a draft PR with recreating it 😞

**Comment by AlexProgrammerDE** [comment]
@LucianBuzzo i have the option in the top right corner under the reviewers to convert this PR into a Draft.

**Comment by tmigone** [comment]
Good call @AlexProgrammerDE

**Comment by AlexProgrammerDE** [comment]
@tmigone but how can you change the design of the website?

**Comment by tmigone** [comment]
@AlexProgrammerDE you can't. Landr does some heuristics so figure out the optimal design for your website depending on what type of information it can find on your repository. This of course means that all websites generated with landr will look very similar to each other, but that is the price you pay for having it autogenerated. This might change or improve in the future as landr adoption grows, but it is what it currently offers.

**Comment by AlexProgrammerDE** [comment]
@balena-ci retst

**Comment by AlexProgrammerDE** [comment]
@balena-ci retest

**Comment by AlexProgrammerDE** [comment]
@tmigone i will import it to the docs.

**Comment by LucianBuzzo** [comment]
landrbot shouldn't be posting so many comments - I'll check out why this is happening

**Comment by AlexProgrammerDE** [comment]
@LucianBuzzo isn't this a normal behaviour? It does this on every commit.

**Comment by LucianBuzzo** [comment]
@AlexProgrammerDE Its meant to post a single comment per PR, then update the link if necessary

**Comment by tmigone** [comment]
bad bot!

**Comment by AlexProgrammerDE** [comment]
@tmigone @LucianBuzzo seems like the bot isnt commenting now.

**Comment by AlexProgrammerDE** [comment]
@LucianBuzzo why are the CHANGELOG.md files in the Landr repo different from the ones in balenaSound?

**Comment by LucianBuzzo** [comment]
@AlexProgrammerDE I pushed a fix for the issue shortly after commenting. WRT to the CHANGELOG, I don't quite understand what you mean, surely it makes sense that the two repos have different changelogs?

**Comment by AlexProgrammerDE** [comment]
@LucianBuzzo i mean the Changelog files are different structured. I mean the date.
balenaSound: https://github.com/balenalabs/balena-sound/blob/master/CHANGELOG.md
Landr: https://github.com/product-os/landr/blob/master/CHANGELOG.md

Is the same versionbot used?

**Comment by AlexProgrammerDE** [comment]
@tmigone do you like the design of the page? I have now added Software and Hardware Requirements.

**Comment by AlexProgrammerDE** [comment]
@LucianBuzzo why are not blog and Changelog options in the navigation?

File: meta.json (hunk 1 above)
Comment (LucianBuzzo) [reply meta.json]: @tmigone Nope, it won't be under source control

**Pull request closed without merging**